asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryTechReport.tex

```diff
@@ -0,0 +1,119 @@
+% LuaLaTeX
+\pdfvariable minorversion=7
+\documentclass[10pt]{article}
+
+% Packages
+\usepackage[no-math]{fontspec}
+\usepackage{geometry}
+\usepackage{hyperref}
+\usepackage[backend=biber]{biblatex} % we use the biber implementation of biblatex for bibliographies
+%\usepackage[nonumberlist, toc]{glossaries}
+\usepackage{graphicx}
+\usepackage{pdfpages}
+\usepackage{booktabs}
+\usepackage{float}
+\usepackage{subcaption}
+\usepackage{fancyhdr}
+\usepackage{amsmath}
+\usepackage{siunitx}
+\usepackage{multirow}
+
+% Formatting
+\geometry{letterpaper, portrait, margin=.85in}
+\defaultfontfeatures{Ligatures=TeX}
+\setmainfont[
+    BoldFont       = Avenir Medium,
+    ItalicFont     = Avenir Book Oblique,
+    BoldItalicFont = Avenir Medium Oblique
+]{Avenir Book}
+\setmonofont{Andale Mono}
+\sisetup{detect-all} % used by siunitx to always typeset units in the font of the current environment
+\urlstyle{same}
+
+\newcommand\theteamname{Midnight Sun Solar Car Team} % should not change normally
+\newcommand\theuniversityname{University of Waterloo} % should also not change normally
+\newcommand\theteamwebsite{www.uwmidsun.com} % should also not normally change
+\newcommand\theteamphone{(519) 888-4567 x32978} % should also not normally change
+
+\newcommand\thetitle{Battery Tech Report} % <--------------- add the title
+\newcommand\thesubtitle{Electrical} % <--------------- add a subtitle or leave the field empty
+\newcommand\theauthor{Minghao Ji} % <--------------- add an author with contact info or comment this line
+\newcommand\theauthorcontact{minghao.ji@uwmidsun.com} % <--------------- add author's email or leave the field empty
+\newcommand\thedate{\today} % <--------------- update the date, use this format
+
+\pagestyle{fancy}
+\renewcommand{\headrulewidth}{0pt}
+\lhead{\thetitle}
+\rhead{\theteamname}
+
+\begin{document}
+
+% Title Page
+\begin{titlepage}
+\large
+\vspace*{2cm}
+\centering
+\includegraphics[width=.25\textwidth]{./figures/midnightSunLogoCircle.png} \\
+\vspace{1.5cm}
+{\LARGE \theteamname} \\
+\theuniversityname \\
+\vspace{2.2cm}
+{\LARGE MSXII} \\
+\vspace{0.4cm}
+{\huge\bfseries \thetitle} \\
+\vspace{0.2cm}
+{\LARGE \thesubtitle} \\
+\vspace{2.2cm}
+\ifdefined \theauthor
+\par Prepared by: \\
+\theauthor \\
+\theauthorcontact \\
+\fi
+\thedate \\
+\vfill
+\theteamwebsite \\
+\theteamphone
+\end{titlepage}
+
+% Main Matter
+\tableofcontents
+%\listoffigures % <-------------- uncomment for list of figures
+%\listoftables % <-------------- uncomment for list of tables
+
+\section{Battery Specifications}
+
+\begin{table}[!htbp]
```

Comment:
For this table maybe break each row ie: `5.2.D.1` into a `\subsection{}` and then use a table for each? It would look much cleaner IMO and allow us to free form text the `5.2.D.8` section without things looking weird. Understandably some sections would be empty but for the string/module you could add a bit about the configuration? IDK if that matters though.

Reply:
donee